lib/jsonapi-serializers/serializer.rb

```diff
@@ -364,6 +364,15 @@ def self.find_recursive_relationships(root_object, root_inclusion_tree, results)
           is_collection = true
           object = serializer.has_many_relationships[unformatted_attr_name]
         end
+
+        if is_valid_attr && attribute_name.include?("_")
+          expected_name = serializer.format_name(attribute_name)
```

Comment:
Ah yes that's true. Can you just remove the is_valid_attr check and move this block below the block below that raises InvalidIncludeError?